Sources/Widgets/Manipulators/LineManipulator/index.js

```diff
@@ -19,6 +20,14 @@ export function projectDisplayToLine(
   const normal = [0, 0, 0];
   vtkMath.cross(lineDirection, viewDir, normal);
   vtkMath.cross(normal, viewDir, normal);
+  // if the active camera viewPlaneNormal and line direction are parallel, no change is allowed
+  const dotProduct = Math.abs(
+    vtkMath.dot(renderer.getActiveCamera().getViewPlaneNormal(), lineDirection)
+  );
+
+  if (1 - dotProduct < EPSILON) {
+    return [];
+  }
```

Comment:
this early return could be done at the beginning of the function

Reply:
Done